core/pages/Home.js

```diff
@@ -28,6 +29,15 @@ export default {
   beforeMount () {
     this.$store.dispatch('category/reset')
   },
+  mounted () {
```

Comment:
This should be moved from core to theme since it's directly calling the ui-specific event